tests/common/fixtures/advanced_reboot.py

```diff
@@ -323,6 +324,11 @@ def __setupTestbed(self):
         logger.info('Copy ARP responder to the PTF container  {}'.format(self.ptfhost.hostname))
         self.ptfhost.copy(src='scripts/arp_responder.py', dest='/opt')
 
+        # Replace fast-reboot script
+        if self.replaceFastRebootScript:
```

Comment:
Can we make this a match condition for 'from' image version? If 'from' image version contains certain wording (e.g. 201803), then proceed with replacing fast-reboot file tagged with same wording?

We might overwrite fast-reboot script according to 'from' image in the future.

Implement the tagging with another PR is fine.

Reply:
OK, will add it in a future pr